pkg/controller/util.go

```diff
@@ -148,6 +149,17 @@ func getSecretName(client kubernetes.Interface, pvc *v1.PersistentVolumeClaim) (
 	return name, nil
 }
 
+// returns the expected image size from the PVC
+func getBlankImageSize(pvc *v1.PersistentVolumeClaim) (string, error) {
+	glog.V(3).Infof("getBlankImage: retrieving requested storage size from pvc \"%s/%s\"\n", pvc.Namespace, pvc.Name)
+	storageSize := pvc.Spec.Resources.Requests[k8sv1.ResourceStorage]
+	iStorageSize := storageSize.Value()
+	// Accounting filesystem overhead - up to 4% according to:
+	// https://rwmj.wordpress.com/2009/11/08/filesystem-metadata-overhead/
+	fStorageSize := float64(iStorageSize) * float64(0.96)
```

Comment:
We don't need to take 96% of the requested size. we will solve on the importer side.